doc/fireplace.txt

```diff
@@ -57,11 +57,19 @@ DOCUMENTATION                                   *fireplace-documentation*
 :Javadoc {class}        Open the java docs for the given class in a browser.
 
                                                 *fireplace-K*
-K                       Look up docs for symbol under cursor.
+K                       Look up (java) doc/spec-form for symbol/keyword under
+                        cursor.
```

Comment:
I found this a tad hard to parse. How about `Look up the doc, javadoc, or spec-form for the identifier under the cursor.`?

Reply:
Sure